frontend/assets/css/iframe.css

```
@@ -4,7 +4,7 @@
 @import url("./iframe/accordion-tabs.css") layer(miyagi);
 @import url("./iframe/jsontree.js.css") layer(miyagi);
 @import url("./iframe/styleguide/index.css") layer(miyagi);
-@import url("./iframe/performance.css") layer(miyagi);
+@import url("./iframe/perf.css") layer(miyagi);
```

Stylelint (17.9.0)

[error] Expected "url("./iframe/perf.css")" to be ""./iframe/perf.css"" (import-notation)

(import-notation)